OrbitGl/TimeGraph.cpp

```diff
@@ -769,32 +756,32 @@ void TimeGraph::SelectAndZoom(const TextBox* text_box) {
   Select(text_box);
 }
 
-//----------------------------------------------------------------------------
-void TimeGraph::OnShiftLeft() {
+void TimeGraph::JumpToBox(JumpDirection jump_direction, JumpScope jump_scope) {
```

Comment:
Make it clear that you are jumping to a neighbor of the selected box. 

Signature could be:

`void JumpToNeighborBox(TextBox* starting_point, JumpDirection..., JumpScope) {}`

Reply:
I would also (as suggested) pass the "starting point" box as an argument.

Reply:
As we are using the word "jump", I would call the variable "from"